src/js/libcs/Operators.js

```diff
@@ -3592,14 +3592,98 @@ evaluator.halfplane$2 = function(args, modifs) {
 //   Geometric elements      //
 ///////////////////////////////
 
-evaluator.allpoints$0 = function(args, modifs) {
+
+// helper for all*(<geoobject>)
+eval_helper.allhelper1arg = function(args, kind, restrict) {
+    if (args.length > 1) return nada;
+    var geoEl = csgeo.csnames[args[0].name];
```

Comment:
Here you are expecting the argument to be a variable, and take its name to access the geometric primitive. That's not the way to go. We need to evaluate the argument, and then deal with the result of that.